changelog/3884.fixed.md

```diff
@@ -0,0 +1 @@
+Remove previously deprecated GET api endpoint "/api/schema/"
```

Comment:
When we create a fragment with towncrier we can have different options

```bash
❯ towncrier create
Issue number (`+` if none): +
Fragment type (security, removed, deprecated, added, changed, fixed):
```

The idea is that we'd use "removed" for something like this instead of "fixed" https://docs.infrahub.app/development/changelog. I think it should be enough to just rename this file.